src/main/java/hudson/plugins/plot/PlotReport.java

```diff
@@ -43,16 +42,16 @@
      */
     private String group;
 
-    public PlotReport(AbstractProject<?, ?> project, String group,
+    public PlotReport(Job<?, ?> job, String group,
```

Comment:
Binary compatibility issue if plugins depend on this one.  Would retain original constructor and have it delegate to a new one with this signature (as mentioned before).